Automattic/wp-calypso
# Help Center: Update tailored articles

Related to https://github.com/Automattic/wp-calypso/issues/88707#issuecomment-2065772920

## Proposed Changes

* Update tailored articles list.

## Testing Instructions



* Use live link and navigate to `settings/general/YOURSITE`
* Open HelpCenter and you should see these articles
 
<img width="414" alt="image" src="https://github.com/Automattic/wp-calypso/assets/2653810/fbe4dbb3-d358-488c-9d2d-880612ca9589">


## Pre-merge Checklist
